init.go

```diff
@@ -0,0 +1,40 @@
+package codacytool
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+)
+
+var (
+	sourceDirFlag           *string
```

Comment:
in `configurationfiles.go` we use a different convention / slightly different syntax to define global variables, I do prefer this way, do you mind to keep it consistent?

Reply:
Removed global variables. Added `RunConfiguration` type, which contains all run configurations and I propagated it into the methods that require it.